Reference/Debugging/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 8.1.0
+---
+
+# Debugging
+In version Umbraco 8.1+ we have enabled a debugging feature from Microsoft & Visual Studio called SourceLink which allows you to step into the native Umbraco CMS source code.
+
+## Enabling SourceLink in Visual Studio 2019
+* Verify that VS Debugging option **SourceLink** checked & **Just My Code** option is unchecked
+
+![Visual Studio 2019 Debug Settings for SourceLink](Images/VS19-enable-sourcelink.png)
+
+## What is SourceLink ?
+You can find details from Microsoft about SourceLink here:
+
+https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink<br/>
```

Comment:
maybe these are better in a bulleted list - with MD links saying where they go... 

eg

- Microsoft Docs
- Scott Hanselman blog post on Sourcelink
- SourceLink on GitHub